src/App.js

```diff
@@ -1,5 +1,25 @@
+import InputManager from './InputManager.js';
+import Game from './Game.js';
+
 class App {
-  async run() {}
+  async run() {
+    const carNames = await this.getCarNames();
+    const attempts = await this.getAttempts();
+    this.playGame(carNames, attempts);
+  }
+
+  async getCarNames() {
+    return InputManager.getCarNames();
+  }
+
+  async getAttempts() {
+    return InputManager.getAttempts();
+  }
+
+  playGame(carNames, attempts) {
+    const game = new Game(carNames, attempts);
+    game.play();
+  }
```

Comment:
`getCarNames`, `getAttempts` 은 단순히 메서드를 호출하는 역할이라서 분리할 필요가 없다고 느껴져요. 아래 제안처럼 직접 호출하는 방식은 어떤가요?

ex. 

```suggestion
    const carNames = await InputManager.getCarNames();
    const attempts = await InputManager.getAttempts();
    const game = new Game(carNames, attempts);
    game.play();
```

Reply:
함수의 분리가 과도하게 되어있는 것 같아서 위 의견에 동의합니다..!

Reply:
확실히 더 깔끔해서 보기 좋네요! 소중한 리뷰 너무나도 감사드립니다!